File: applications/src/main/java/com/mydemo/applications/service/PositionService.java
package com.mydemo.applications.service;

import com.mydemo.applications.entity.JzPosition;
import com.mydemo.applications.entity.PositionPojo;
import com.mydemo.applications.entity.PositionType;
import com.mydemo.applications.entity.SearchEntity;

import java.util.List;
import java.util.Map;

public interface PositionService {
    public int addPosition(PositionPojo positionPojo);
    public List<PositionType> findTypePid(Integer pid);
    public Map findDesc(Integer id,Integer userId);
    public List<JzPosition> findAllPosi(SearchEntity entity);
    public List<PositionType> findAllType();
    public int addgunzhu(Integer descId,Integer userId);
    public int remguanzhu(Integer descId,Integer userId);
    public int addPublish(Integer id);
    public int remPublish(Integer id);
    public PositionPojo findOne(Integer id);
    public int update(PositionPojo positionPojo);
}

File: applications/src/main/resources/templates/js/service/positionService.js
app.service('positionService',function($http){
    this.findTypePid=function (pid) {
        return $http.get('../position/findTypePid?pid='+pid);
    }
    this.add=function (entity) {
        return $http.post('../position/addPosition',entity);
    }
    this.update=function (entity) {
        return $http.post('../position/update',entity);
    }
    this.findOne=function (id) {
        return $http.get('../position/findOne?id='+id);
    }
})